tests/test-cases/complete.yaml

```diff
@@ -12,8 +12,7 @@ tests:
     expect:
       diff: []
       stderr:
-        - 'ERRO ATMOS_SHLVL \(11\) exceeds maximum allowed depth \(10\)'
-        - "ERRO exit status 1"
+        - 'FATA ATMOS_SHLVL exceeds maximum allowed depth\. Infinite recursion\? current=(\d+), max=(\d+)\n(?:FATA exit status 1\n)+'
```

Comment:
This regex seems a lot more complicated when we already support multi regex. We don't need to solve it in one expression.